clients/pom.xml

```diff
@@ -0,0 +1,84 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <name>lakefs-parent</name>
+  <groupId>io.lakefs</groupId>
+  <artifactId>lakefs-parent</artifactId>
+  <version>0</version>
+  <packaging>pom</packaging>
+
+  <description>lakeFS root package for POM inheritance.</description>
+  <url>https://github.com/treeverse/lakeFS/tree/master/clients</url>
+
+  <licenses>
+    <license>
+      <name>apache2</name>
+      <url>http://www.apache.org/licenses/</url>
+      <distribution>repo</distribution>
+    </license>
+  </licenses>
+
+  <developers>
+    <developer>
+      <name>Treeverse lakeFS dev</name>
+      <email>support@treeverse.io</email>
+      <organization>treeverse.org</organization>
+      <organizationUrl>https://treeverse.org</organizationUrl>
+    </developer>
+  </developers>
+
+  <scm>
+    <url>https://github.com/treeverse/lakefs</url>
+    <connection>scm:git:git://github.com/treeverse/lakefs.git</connection>
+    <developerConnection>scm:git:ssh://git@github.com/treeverse/lakefs.git</developerConnection>
+  </scm>
+
+  <profiles>			<!-- to deploy (signed), must run "gpg -Prelease deploy"-->
```

Comment:
I would expect the deploy command to use the profile id you defined below. i.e. "gpg -Ptreeverse-signing deploy"

Reply:
Indeed you can do that. Not sure where this profile selection is missing, so will add it when I discover.

Thanks!